File: lab1/src/main/java/com/lab1/lab1/model/model/Person.java
package com.lab1.lab1.model.model;

public class Person {
    private String name; //Поле не может быть null, Строка не может быть пустой
    private Color eyeColor; //Поле может быть null
    private Color hairColor; //Поле может быть null
    private Location location; //Поле может быть null
    private String passportID; //Длина строки не должна быть больше 34, Значение этого поля должно быть уникальным, Поле может быть null
    private Country nationality; //Поле не может быть null
}

File: frontend/lib/features/main/domain/entities/person.dart
class Person {
  String name;
  Color eyeColor;
  Color hairColor;
  Location location;
  String passportID;
  Country nationality;

  Person({
    required this.name,
    required this.eyeColor,
    required this.hairColor,
    required this.location,
    required this.passportID,
    required this.nationality,
  });
}

class Location {
  double x;
  int y;
  int z;

  Location({
    required this.x,
    required this.y,
    required this.z,
  });
}

enum Color {
  RED,
  BLACK,
  WHITE;
}

enum Country {
  RUSSIA,
  INDIA,
  JAPAN;
}
